minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloClusterImpl.java

```diff
@@ -608,6 +618,8 @@ public synchronized void start() throws IOException, InterruptedException {
 
   private void verifyUp() throws InterruptedException, IOException {
 
+    int numTries = 10;
+
     requireNonNull(getClusterControl().managerProcess, "Error starting Manager - no process");
     requireNonNull(getClusterControl().managerProcess.info().startInstant().get(),
```

Comment:
I am not sure this error message will get thrown. If you look at the impl of `Optional.get()` it will throw its own exception:
<pre>
 public T get() {
        if (value == null) {
            throw new NoSuchElementException("No value present");
        }
        return value;
    }
</pre>

Reply:
Yeah, it's a redundant check with optional.

Reply:
We should change these to just check `Optional.isEmpty()` and throw an exception. The message "No value present" is not very helpful.

Reply:
I have a fix incoming that will also address #2431